src/components/POLine/Item/ItemForm.js

```diff
@@ -1,7 +1,16 @@
 import React, { Component } from 'react';
 import { FormattedMessage } from 'react-intl';
 import PropTypes from 'prop-types';
-import { Field } from 'redux-form';
+import {
+  Field,
+  getFormValues,
```

Comment:
I think you can pass form values from PO Line form

Reply:
In this case, we'll get not updated form values as prop.